tool.sh

```diff
@@ -46,6 +47,8 @@ elif [ "$1" == "prod" ]; then
   readme
   # build and upload package to prod pypi
   python setup.py sdist upload -r pypi
+elif [ "$1" == "update" ]; then
+  git submodule foreach git pull origin master
```

Comment:
What if we add more submodules in the future in different location? Do we want `update` to fetch all of them?